api/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10 AS builder
+
+ARG TARGETARCH
+USER root
+RUN microdnf install -y tar gzip unzip
+
+# install platform specific go version
+RUN curl -O -J  https://dl.google.com/go/go1.22.0.linux-${TARGETARCH}.tar.gz
+RUN tar -C /usr/local -xzf go1.22.0.linux-${TARGETARCH}.tar.gz
+RUN ln -s /usr/local/go/bin/go /usr/local/bin/go
+
+
+# Install protoc
+RUN curl -L -o protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v27.1/protoc-27.1-linux-x86_64.zip \
```

Comment:
I should have said this before, but could we pin to version `v25.1` for now, because this is currently the latest version of protoc that is supported by the latest version of protoc-gen-grpc-java. :/

Versions given [here](https://github.com/grpc/grpc-java/tree/master) for java, but I can't find the reference where it says why it needs to be these versions.

Info on weird new versioning scheme: https://protobuf.dev/support/version-support/

Reply:
No problem with that, I'll go ahead and update the version.